### ISSUE TITLE
Fine tuning models?

### ISSUE THREAD

**Issue by youssefavx** [speechbrain/speechbrain#568]
Hi, are there usage examples for how to fine tune the huggingface models (e.g. speech recognition and speech enhancement) based on our own datasets? 

I have a dataset of noisy audio from a speaker and I'd like to transcribe them but I'm thinking of fine tuning on transcriptions with the type of noise that occurs in my dataset to increase accuracy.

**Comment by youssefavx** [comment]
Also thank you for being of the few who actually understand that it needs to be as simple as one button!

```
from speechbrain.pretrained import TransformerASR

asr_model = TransformerASR.from_hparams(source="speechbrain/asr-transformer-transformerlm-librispeech", savedir="pretrained_models/asr-transformer-transformerlm-librispeech")
asr_model.transcribe_file("speechbrain/asr-transformer-transformerlm-librispeech/example.wav")
```

**Comment by TParcollet** [comment]
Hi happy to see your interest :) 

Have a look here (last section): https://colab.research.google.com/drive/1LN7R3U3xneDgDRK2gC5MzGkLysCWxuC3?usp=sharing If this isn't enough we can expand more.

**Comment by youssefavx** [comment]
Amazing! Really appreciate the tremendous effort you guys put into this. I'll give this a shot and report back!

**Comment by leijue222** [comment]
Hi, @TParcollet 
Which languages does the ASR currently provided by SpeechBrain support?
I want to test the pre-trained model you provided to see the accuracy and then consider whether to use finetune on the language-corresponding dataset to improve the accuracy.
